crates/astria-composer/src/executor/builder.rs

```diff
@@ -0,0 +1,74 @@
+use std::time::Duration;
+
+use astria_core::sequencer::v1::{
+    transaction::action::SequenceAction,
+    Address,
+};
+use astria_eyre::{
+    eyre,
+    eyre::{
+        eyre,
+        Context,
```

Comment:
`Context` is a compatibility trait used for migrating anyhow to eyre and will soon be feature-gated:
```suggestion
        WrapErr as ),
```